src/main/java/uk/gov/pay/adminusers/model/CreateUserRequest.java

```diff
@@ -54,7 +54,7 @@ public static CreateUserRequest from(JsonNode node) {
                 gatewayAccountIds =
                         ImmutableList.copyOf(node.get(FIELD_GATEWAY_ACCOUNT_IDS).iterator())
                                 .stream().map(JsonNode::asText)
-                                .sorted(Comparators.usingNumericComparator())
+                                .sorted(Comparators.compareGatewayAccounts())
```

Comment:
`Comparators.numericallyThenLexicographically()`?

Reply:
that is the level of precision I'd expect from you, David :D